__tests__/task.test.js

```diff
@@ -33,6 +33,28 @@ describe('Task', () => {
     });
   });
 
+  describe('get', () => {
```

Comment:
same as above

Reply:
and what about https://github.com/alex-popov-tech/backend-project-lvl4/blob/de12aa0f6ec63fd386a3b14c2d92e1e5503cfa13/server/routes/tasks.js#L20